plugins/generic/metadataExport/MetadataExportBlockPlugin.inc.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file plugins/generic/metadataExport/MetadataExportBlockPlugin.inc.php
+ *
+ * Copyright (c) 2013-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class MetadataExportBlockPlugin
+ * @ingroup plugins_generic_metadataExport
+ *
+ * @brief Class for block component of metadata export plugin
+ */
+
+import('lib.pkp.classes.plugins.BlockPlugin');
+
+class MetadataExportBlockPlugin extends BlockPlugin {
+	
+	/* @var parentPluginName String - name of the parent plugin object */
+	var $parentPluginName;
+	
+	/* @var metadataExportPlugins Array - all plugins of the metadataexport category */
+	var $metadataExportPlugins;
+
+	
+	/**
+	 * @function MetadataExportBlockPlugin - Constructor
```

Comment:
For consistency elsewhere, don't use the `@function` directive; just describe the function in the first line. (We're using Doxygen for self-documentation.)